SqlTrainer.UserService.Infrastructure/BusinessLogics/UserBusinessLogic.cs

```diff
@@ -12,4 +12,17 @@ public UserBusinessLogic(IUserRepository repository, IPasswordHasher passwordHas
         this.repository = repository;
         this.passwordHasher = passwordHasher;
     }
+
+    public async Task<IOperationResult<Guid>> AddAsync(User model)
+    {
+        var hashedPassword = passwordHasher.Hash(model.HashedPassword).Result;
```

Comment:
Please check if OperationResultState is OK. Because when you try to get Result when state is BadFlow you Exception will be thrown

Reply:
ok